src/vue/modules/asset-explorer/store/index.js

```diff
@@ -0,0 +1,72 @@
+import { Balance } from '../wrappers/balance'
+import { Asset } from '../wrappers/asset'
+
+import { api } from '../_api'
+import { types } from './types'
+
+import { base } from '@tokend/js-sdk'
+
+export const state = {
+  assets: [],
+  balances: [],
+
+  balancesOwnerId: '',
+}
+
+export const mutations = {
+  [types.SET_ASSETS] (state, assets) {
+    state.assets = assets
+  },
+  [types.SET_NEXT_ASSETS] (state, assets) {
+    state.assets = state.assets.concat(assets)
+  },
+  [types.SET_BALANCES] (state, balances) {
+    state.balances = balances || []
+  },
+  [types.SET_BALANCES_OWNER_ID] (state, id) {
+    state.balancesOwnerId = id
```

Comment:
not sure about it, the back-end-guys call it `address`, we call it `account id`. here we define another one name for account id. I would rather stick to extracting it from the wallet or renaming it to `accountId` or `balanceId`